src/main/java/seedu/txtdata/TxtFileStatus.java

```diff
@@ -74,80 +82,34 @@ public static void initializeExpenditureList(ExpenditureList expenditures) throw
         Scanner s = new Scanner(f); // create a Scanner using the File as the source
         while (s.hasNext()) {
             String saveString = s.nextLine();
-            String[] saveData = saveString.split("d/|v/|t/|p/|n/|o/");
-            switch (saveData[0]) {
+            String[] saveData = saveString.split("d/|v/|t/|p/|n/|o/|r/");
+            switch (saveData[INDEX_TYPE]) {
```

Comment:
good implementation, ensures that it does not automatically unmark whenever user starts the app